docs/sources/oncall-api-reference/outgoing_webhooks.md

```diff
@@ -10,7 +10,7 @@ weight: 700
 > endpoint remains available and is compatible with previous callers but under the hood it will interact with the
 > new webhooks objects.  It is recommended to use the /webhooks endpoint going forward which has more features.
 
-For more details about specific fields of a webhook see [outgoing webhooks][outgoing-webhooks] documentation.
+For more details about specific fields of a webhook see [outgoing webhooks](../../outgoing-webhooks) documentation.
```

Comment:
This indeed fixes an erroneous sentence (https://grafana.com/docs/oncall/latest/oncall-api-reference/outgoing_webhooks/#:~:text=%5Boutgoing%20webhooks%5D%5Boutgoing%2Dwebhooks%5D) but
the actual broken link in this page reported by the crawler is https://github.com/grafana/oncall/blob/d6d11b723880cec74dc34bd0db6fe7a4759287c7/docs/sources/oncall-api-reference/outgoing_webhooks.md?plain=1#L108